src/components/layout/inlineBlockList.stories.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, boolean, select } from '@storybook/addon-knobs';
+
+import InlineBlockList from './InlineBlockList';
+import Chip from '../interactive/Chip';
+import README from './InlineBlockList.md';
+
+const noop = () => {};
+const parentStyle = {
+  height: '100vh',
+  width: '100vw',
+};
+
+storiesOf('Layout/InlineBlockList', module)
+  .addDecorator(withKnobs)
+  .add(
+    'Knobs',
+    () => (
+      <div className="alignChild--center--center" style={parentStyle}>
+        <InlineBlockList
+          separator={text('separator', '✈️')}
+          items={['New York', 'Paris', 'Schenectady', 'Los Angeles', 'Hong Kong']}
+        />
+      </div>
+    ),
+    { notes: { markdown: README } }
+  )
+  .add('Chips list', () => (
+    <InlineBlockList
+      items={[
+        <Chip size="lg" onClose={noop} label="Horses" theme="outline" />,
+        <Chip size="lg" onClose={noop} label="Wombats" theme="outline" />,
+        <Chip size="lg" onClose={noop} label="Seals" theme="outline" />,
+        <Chip size="lg" onClose={noop} label="Dogs" theme="outline" />,
```

Comment:
are the `noop`s being passed in here because some Chip functionality fails without them?